MessageManager.class.nut

```diff
@@ -86,12 +88,15 @@ class MessageManager {
     // Max number of auto retries
     _maxAutoRetries = null
 
+    // First message ID
+    _firstMessageID = null
```

Comment:
I don't really think we need a class-level variable here. This configuration setting is used only from the constructor.